### PR TITLE
Product Gallery > Pager block: Add E2E tests for the block

### PR DESCRIPTION
**Pull request by thealexandrelara**
## What
This PR adds E2E tests for the Pager block

## Testing Instructions

_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Ensure that the E2E tests pass on the CI.

* [x] Do not include in the Testing Notes 
* [x] Should be tested by the development team exclusively 

## WooCommerce Visibility

Required:

* [ ] WooCommerce Core
* [ ] Feature plugin
* [ ] Experimental
* [x] N/A

### CODE REVIEW

File: tests/e2e/tests/product-gallery/inner-blocks/product-gallery-pager/product-gallery-pager.block_theme.side_effects.spec.ts [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+import { ProductGalleryPage } from '../../product-gallery.page';
+
+const blockData = {
+	name: 'woocommerce/product-gallery-pager',
+	selectors: {
+		frontend: {
+			pagerBlock:
+				'div[data-block-name="woocommerce/product-gallery-pager"]',
+			pagerListContainer:
+				'ul.wp-block-woocommerce-product-gallery-pager__pager',
```
Comment (danieldudzic) [inline tests/e2e/tests/product-gallery/inner-blocks/product-gallery-pager/product-gallery-pager.block_theme.side_effects.spec.ts]: `trunk` already contains the migrated the Pager classnames `wc-block-product-gallery-pager__pager` - let's make sure update them in the E2E tests too.

**Review by danieldudzic — APPROVED** [review]
All 5 tests passed for me locally! ✅ 

I'd maybe add, to the testing instructions, the specific command to run the tests.

`npm run test:e2e:side-effects ./tests/e2e/tests/product-gallery/inner-blocks/product-gallery-pager/product-gallery-pager.block_theme.side_effects.spec.ts`

Thanks for working on those tests!

File: tests/e2e/tests/product-gallery/inner-blocks/product-gallery-pager/product-gallery-pager.block_theme.side_effects.spec.ts (hunk 1 above)
Comment (thealexandrelara) [reply tests/e2e/tests/product-gallery/inner-blocks/product-gallery-pager/product-gallery-pager.block_theme.side_effects.spec.ts]: Thank you for pointing this out, I just updated that 🙌
